src/content/docs/apis/lookups-service-api/lookups-service-api.mdx

```diff
@@ -10,31 +10,36 @@ redirects:
   - /docs/apis/lookup-service-api/lookup-service-api
 ---
 
-Use the NRQL Lookups API to create and manage [lookup tables](/docs/logs/ui-data/lookup-tables-ui/#overview).
+If you have custom data that's not reported to New Relic out of the box, you likely use [lookup tables](/docs/logs/ui-data/lookup-tables-ui/#overview) to query certain kinds of telemetry data. With our NRQL lookups API, you can programmatically create, update, or delete your lookup tables. This is a good option for users who want a more efficient way to query and handle their custom data.  
 
-## Before you start
-
-The NRQL Lookups API is a REST API that allows you to manage lookup tables programmatically. As another option you can also [manage lookup tables through our UI](/docs/logs/ui-data/lookup-tables-ui/#lookup-ui).
+The NRQL lookups API is a REST API that allows you to manage lookup tables programmatically. If you don't want to work with REST APIs, you can [manage lookup tables via our UI](/docs/logs/ui-data/lookup-tables-ui/#lookup-ui).
```

Comment:
I'm not sure about the "If you don't want to work with the REST APIs" line. I think the original "As another option" is better here.  Or maybe something like "For simple, non-programatic Lookup Table maintenance, you can manage..".

At the very least, remove the "s" on "APIs".